testcode/lib/joy-test.c

```diff
@@ -5,14 +5,29 @@
 #include <conio.h>
 #include <joystick.h>
 
-
 #ifdef JOYSTICK_DRIVER
 
 /* A statically linked driver was named on the compiler's command line.
 ** Make sure that it is used instead of a dynamic one.
 */
 #  undef DYN_DRV
 #  define DYN_DRV       0
+
+/*
+ * link existing drivers like this:
```

Comment:
Could you please adjust the comment style on this and the following lines to match the exsisting style on line 11?